submissions/ayokofemi/Database 1/init.sql

```diff
@@ -0,0 +1,44 @@
+--for the foodTable--
+CREATE TABLE food (
+id INT NOT NULL AUTO_INCREMENT,
+type VARCHAR(20) NOT NULL,
+PRIMARY KEY (id)
+);
+
+--for the vendorFood--
+CREATE TABLE vendor_food (
+id INT NOT NULL AUTO_INCREMENT,
+admin_id INT NOT NULL,
+food_id INT NOT NULL,
+name VARCHAR(30) NOT NULL,
+amount FLOAT(4,2) NOT NULL,
+PRIMARY KEY (id)
+);
+
+--for the adminsTable--
+CREATE TABLE admins (
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(30) NOT NULL,
+type  VARCHAR(16) NOT NULL,
```

Comment:
This should be ENUM ("1", "2", "3", "4") that will be specific that we have 4 types